Multisig/Logic/Models/Network.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  Chain.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/23/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension Network {
```

Comment:
all of these methods should be tested in the "integration tests" target - we have similar database tests there

Reply:
I'll do that